anchore_engine/services/policy_engine/engine/feeds/feeds.py

```diff
@@ -653,25 +736,18 @@ def record_count(self, group_name: str, db: Session) -> int:
         """
         return self._find_match(db).count()
 
-    def _flush_group(
-        self, group_obj: FeedGroupMetadata, operation_id: Optional[str] = None
-    ) -> None:
+    def _flush_group(self, group_obj: FeedGroupMetadata) -> None:
         """
         Flush a specific data group. Do a db flush, but not a commit at the end to keep the transaction open.
```

Comment:
Nit: Also not you change, but where does the transaction finally get completed at?

Reply:
The naming of this is confusing. _flush_group actually deletes the group. But in sqlalchemy, flush actually means `Flush all the object changes to the database.`

Reply:
exception handling needs to be cleaned up here in the future